x-pack/plugin/src/test/resources/rest-api-spec/api/repositories.clear_metering_archive.json

```diff
@@ -0,0 +1,30 @@
+{
+  "repositories.clear_metering_archive":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/master/clear-repositories-metering-archive-api.html",
+      "description":"Removes the archived repositories metering information present in the cluster."
+    },
+    "stability":"experimental",
+    "url":{
+      "paths":[
+        {
+          "path":"/_nodes/{node_id}/_repositories_metering/{max_version_to_clear}",
```

Comment:
```suggestion
          "path":"/_nodes/{node_id}/_repositories_metering/{version}",
```

Reply:
I'll prefer to leave that parameter name, as it clears up to the specified version.